_episodes/12-cluster.md

```diff
@@ -23,143 +27,278 @@ On the first day, her supervisor greets her friendly and welcomes her to the job
+
+~~~
+$ free -g
+~~~
+{: .language-bash}
+
+> ## Units and Language
+> 
+> A computer's memory and disk are measured in units called *Bytes* (one Byte are bits). As today's files and memory have grown to be large given historic standards, volumes are noted using the [SI](https://en.wikipedia.org/wiki/International_System_of_Units) prefixes. So 1000 Bytes is a Kilobyte (kB), 1000 Kilobytes is a Megabyte, 1000 Megabytes is a Gigabyte etc. 
+> 
+> History and common language have however mixed this notation with a different meaning. When people say "Kilobyte", they mean 1024 Bytes instead. In that spirit, a Megabyte are 1024 Kilobytes. To address this ambiguity, the [International System of Quantities](https://en.wikipedia.org/wiki/International_System_of_Quantities) standardizes the binary prefixes (with base of 1024) by the prefixes kibi, mibi, gibi, etc. For more details, see [here](https://en.wikipedia.org/wiki/Binary_prefix)
+{: .callout}
+
+> ## Relative to your Laptop
+> 
+> Note down the number of CPU cores, the amount of RAM and the total disk space available on `{{ site.workshop_login_host }}`. Compare it to your laptop!
+> 
+> Bonus: Divide the values obtained from `{{ site.workshop_login_host }}` by the numbers obtained for your laptop. How much more powerful is the login node of the cluster compared to your laptop?
 {: .challenge}
 
-> ## Explore The Head Node
->
-> Now we'll compare the size of your computer with the size of the head node: To see the number of
-> processors, run:
->
-> ```
-> nproc --all
-> ```
-> {: .bash}
->
-> or
+
+## Transferring Data
+
+The admin continues to explain, that typically people perform computationally heavy tasks on the cluster and prepare files that contain the results or a subset of data to create final results on the individuals laptop. So communication to and from the cluster is done mostly by transferring files. For example, Lola is asked to use a [file of her liking]({{page.root}}/filesystem/home/admin/this_weeks_canteen_menus/todays_canteen_menu.pdf) and transfer it over. For this, he advises her to use the secure copy command, `scp`. As before, this establishes a secure encrypted temporary connection between Lola's laptop and the cluster just for the sake of transferring the files. After the transfer has completed, scp will close the connection again.
+
+~~~ 
+$ scp todays_canteen_menu.pdf lola@{{ site.workshop_login_host }}:todays_canteen_menu.pdf
+~~~
+{: .language-bash}
+
+~~~ 
+todays_canteen_menu.pdf                                              100%   28KB  27.6KB/s   00:00
+~~~
+{: .output}
+
+She can now `ssh` into the cluster again and check, if the file has arrived after she just uploaded it:
+
+~~~ 
+$ ssh lola@{{ site.workshop_login_host }}
+Last login: Tue Mar 14 14:17:44 2017 from lolas_laptop
+-bash-4.1$ ls
```

Comment:
change of prompt, prefix `-bash-4.1` before the `$`

Reply:
I opted for `$` with 8706039. Very good catch! Thanks.